shared/shot.js

```diff
@@ -374,7 +374,9 @@ class AbstractShot {
     // eslint-disable-next-line no-control-regex
     filenameTitle = filenameTitle.replace(/[:\\<>/!@&?"*.|\x00-\x1F]/g, " ");
     filenameTitle = filenameTitle.replace(/\s{1,4000}/g, " ");
-    let clipFilename = `Screenshot-${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()} ${filenameTitle}`;
+    let today = new Date();
+    let filenameDate = new Date(today.getTime() - today.getTimezoneOffset() * 60 * 1000).toISOString().substring(0,10);
```

Comment:
whoops! should have commented on the whole PR, not that (now outdated) commit. Note that the line you replaced uses `date`, which is based on the `createdDate` of the shot (line 373 above). You'll want to use that date for generating the `filenameDate`.

Reply:
Sure!